### PR TITLE
document allow-substitutes limitation

### PR DESCRIPTION
**Pull request by Mic92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,17 @@ in drvs
...
 
+### Packages with `allowSubstitutes = false`
+
+If your package set you are building has packages at the top level scope that
+have the attribute `allowSubstitutes = false;` set, than `nix-build-uncached`
+will build/download them everytime. A workaround is to use the following
+[gist](https://gist.github.com/7bc9210b1c4d0afe390f0d425b50e02f):
+
+```nix
+# TODO example
```
Comment (Mic92) [inline README.md:89]: @zimbatm do you have an example I can put here?

File: README.md [hunk 2]
```diff
@@ -78,6 +78,17 @@ in drvs
 $ nix-build-uncached ./ci.nix
 ```
 
+### Packages with `allowSubstitutes = false`
+
+If your package set you are building has packages at the top level scope that
+have the attribute `allowSubstitutes = false;` set, than `nix-build-uncached`
+will build/download them everytime. A workaround is to use the following
+[gist](https://gist.github.com/7bc9210b1c4d0afe390f0d425b50e02f):
```
Comment (zimbatm) [inline README.md:86]: Feel free to embed the script in the repo if you want.

File: README.md (hunk 1 above)
Comment (zimbatm) [reply README.md:89]: I expect the most common one to be `pkgs.writeScriptBin` or one of the other trivial builders. All my examples are private right now.

File: README.md (hunk 2 above)
Comment (Mic92) [reply README.md:86]: It is not really usable from the nix store. People should rather vendor it. Maybe I put it in the wiki.

**Pull request merged**